### PR TITLE
making changes suggested by @RLumSK in PR #9

### PR DESCRIPTION
**Pull request by SteveGrehl**
I think this PR addresses the needs better than PR #9.

### CODE REVIEW

File: meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{% set version = "0.8" %}
```
Comment (RLumSK) [inline meta.yaml:1]: Why 0.8?

File: src/xlum/__init__.py
Comment (RLumSK) [inline src/xlum/__init__.py:1]: Why here `.importer`? Sorry, my knowledge on Python is limited

**Review by RLumSK — COMMENTED** [review]
I've got just two questions (see comments) the rest is fine with me!

File: src/xlum/__init__.py
Comment (SteveGrehl) [reply src/xlum/__init__.py:1]: It's a relative import on the package level. See: https://peps.python.org/pep-0328/

File: meta.yaml (hunk 1 above)
Comment (SteveGrehl) [reply meta.yaml:1]: In preparation for our conda release, we increased our version in \_\_init\_\_.py. See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html for more information.

**Comment by SteveGrehl** [comment]
@RLumSK can you approve PR #10?

File: meta.yaml (hunk 1 above)
Comment (RLumSK) [reply meta.yaml:1]: OK, but I thought it is 0.0.8 for xlum?

**Review by RLumSK — APPROVED** [review]

**Comment by RLumSK** [comment]
@SteveGrehl I approved regardless of the open 0.8 or 0.0.8 question

File: meta.yaml (hunk 1 above)
Comment (SteveGrehl) [reply meta.yaml:1]: That's correct :)

**Comment by SteveGrehl** [comment]
Fixed open question and published __v0.0.8__ here; https://pypi.org/project/xlum/

**Pull request merged**